kategory/src/test/kotlin/kategory/data/EitherTTest.kt

```diff
@@ -165,27 +165,27 @@ class EitherTTest : UnitSpec() {
             forAll { a: Int ->
                 val x = { b: Int -> EitherT.pure<Id.F, Int, Int>(b * a) }
                 val option = EitherT.pure<Id.F, Int, Int>(a)
-                option.flatMap(x) == EitherTInstances<Id.F, Int>(Id).flatMap(option, x)
+                option.flatMap(x) == EitherT.monad<Id.F, Int>(Id).flatMap(option, x)
             }
         }
 
         "EitherTMonad#tailRecM should execute and terminate without blowing up the stack" {
             forAll { a: Int ->
-                val value: EitherT<Id.F, Int, Int> = EitherTInstances<Id.F, Int>(Id).tailRecM(a) { b ->
+                val value: EitherT<Id.F, Int, Int> = EitherT.monad<Id.F, Int>(Id).tailRecM(a) { b ->
                     EitherT.pure<Id.F, Int, Either<Int, Int>>(Either.Right(b * a))
-                }
+                }.ev().ev()
```

Comment:
These two `ev()` may be causing an exception and why Travis stopped working after the previous test.

```
kategory.EitherTTest > EitherTTest.EitherTMonad#flatMap should be consistent with EitherT#flatMap PASSED
:kategory:test FAILED 
```

Seems like something for the Kotlin team.